source/guides/deployment.rst

```diff
@@ -21,6 +21,7 @@ These guides will get you up and running with Mattermost in minutes.
 
 * :doc:`Quick install guide </getting-started/light-install>` - Deploy in minutes via Mattermost Omnibus on Ubuntu.
 * :doc:`Run Mattermost via Docker </install/setting-up-local-machine-using-docker>` - Launch a Mattermost server instantly to test functionality and build integrations.
+* :doc:`Install Mattermost from Tar </install/install-tar>` - Install Mattermost server from a compressed tarball.
```

Comment:
You'll need to add this new page to the toctree above to surface it up via the LHS navigation pane.